common/src/main/java/com/microsoft/identity/common/internal/fido/FidoUtil.kt

```diff
@@ -0,0 +1,70 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.common.internal.fido
+
+import android.os.Build
+import com.microsoft.identity.common.java.constants.FidoConstants.Companion.WEBAUTHN_QUERY_PARAMETER_FIELD
+import com.microsoft.identity.common.java.constants.FidoConstants.Companion.WEBAUTHN_QUERY_PARAMETER_VALUE
+import com.microsoft.identity.common.java.logging.Logger
+import java.util.AbstractMap
+
+/**
+ * A utility class to help with FIDO related operations.
+ */
+class FidoUtil {
```

Comment:
Since this is a Util class, do we still need a class for this? Why not write a top level function?

Reply:
Put the method in AndroidPlatformUtil: 3869563cc6c9c5e871ab4cff27942a95bd667e98
(Hopefully that's the top level class you were thinking of)

Reply:
My comment was more around considering writing a class-less kotlin file with just functions but I guess that wasn't going to be possible anyway since we need interop with java